python/lsst/pipe/base/execution_reports.py

```diff
@@ -257,11 +257,13 @@ def to_summary_dict(self, butler: Butler, logs: bool = True) -> dict[str, Any]:
         failed_quanta = {}
         for node_id, log_ref in self.failed.items():
             quantum_info: dict[str, Any] = {"data_id": log_ref.dataId.byName()}
-            if logs:
+            if do_store_logs:
                 try:
                     log = butler.get(log_ref)
                 except LookupError:
                     quantum_info["error"] = []
+                except FileNotFoundError:
+                    quantum_info["error"] = None
```

Comment:
It might be worth printing a message here that you did miss some files just so people know that it happened, unless that's dealt with elsewhere.